pkg/oc/cli/logs/logs_test.go

```diff
@@ -91,12 +92,13 @@ func TestRunLogForPipelineStrategy(t *testing.T) {
 		o := &LogsOptions{
 			IOStreams: streams,
 			KubeLogOptions: &kcmd.LogsOptions{
+				IOStreams: streams,
 				Object:    tc.o,
 				Namespace: "foo",
 			},
 			Client: fakebc.Build(),
 		}
-		if err := o.RunLog(); err != nil {
+		if err := o.runLogPipeline(); err != nil {
```

Comment:
Why this change?

Reply:
The test `TestRunLogForPipelineStrategy` was only testing that portion of the code. Splitting it out made it easier to test